collections/profile.js

```diff
@@ -375,7 +375,7 @@ CollegeProfileSchema = new SimpleSchema({
 
   "intentions": {type: _intentionSchema, optional: true},
   "recommendations": {type: _recommendationSchema, optional: true},
-
+  "subscribedToNewsletter": {type: Boolean, optional: true },
```

Comment:
You probably don't need this here, unless I'm missing something. `subscribedToNewsletter` exists on the `profile` object of the Users document, but this is the CollegeProfile schema (different things).

`profile` on the Users document is blackboxed, so it doesn't define the `subscribedToNewsletter` field explicitiy in the schema. It actually comes from Telescope's schema (which we should probably do a better job of keeping in sync with our app).